hedgehog/src/Hedgehog.hs

```diff
@@ -165,7 +168,7 @@ import           Hedgehog.Internal.Property (classify, cover)
 import           Hedgehog.Internal.Property (discard, failure, success)
 import           Hedgehog.Internal.Property (DiscardLimit, withDiscards)
 import           Hedgehog.Internal.Property (eval, evalM, evalIO)
-import           Hedgehog.Internal.Property (evalEither, evalExceptT)
+import           Hedgehog.Internal.Property (evalMaybe, evalMaybeM, evalEither, evalEitherM, evalExceptT)
```

Comment:
It'd be nice to have those in alphabetical order.

Reply:
sorted evalMaybeXyz after evalEither